### PR TITLE
Stop relying on symfony/symfony replace section

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: src/Flex.php [hunk 1]
```diff
@@ -108,17 +108,20 @@ public function activate(Composer $composer, IOInterface $io)
...
+        $setRepositories = \Closure::bind(function (RepositoryManager $manager) use (&$symfonyRequire, $downloader) {
             $manager->repositoryClasses = $this->repositoryClasses;
             $manager->setRepositoryClass('composer', TruncatedComposerRepository::class);
             $manager->repositories = $this->repositories;
             $i = 0;
             foreach (RepositoryFactory::defaultRepos(null, $this->config, $manager) as $repo) {
                 $manager->repositories[$i++] = $repo;
                 if ($repo instanceof TruncatedComposerRepository && $symfonyRequire) {
-                    $repo->setSymfonyRequire($symfonyRequire, $this->io);
+                    $versions = $downloader->get('/versions.json')->getBody();
```
Comment (nicolas-grekas) [inline src/Flex.php]: we could put this outside of the loop

File: src/Cache.php [hunk 2]
```diff
@@ -51,19 +53,16 @@ public function removeLegacyTags(array $data): array
             return $data;
         }
 
-        $symfonyPackages = [];
+        $masterVersion = $this->versions['master'];
         $symfonySymfony = $data['packages']['symfony/symfony'];
-
```
Comment (nicolas-grekas) [inline src/Cache.php]: could be kept :)

File: src/Flex.php (hunk 1 above)
Comment (fabpot) [reply src/Flex.php]: Done

File: src/Cache.php (hunk 2 above)
Comment (fabpot) [reply src/Cache.php]: Done

**Comment by nicolas-grekas** [comment]
Looks like we missed something, this break the Symfony CI :)
https://travis-ci.org/symfony/symfony/builds/540305191

**Comment by nicolas-grekas** [comment]
Maybe because 5.0 is not listed here?
https://flex.symfony.com/versions.json

**Comment by fabpot** [comment]
I've just updated `versions.json`.

**Comment by fabpot** [comment]
@nicolas-grekas I'm going to have a look at this issue this morning.

**Pull request merged**